streamer/src/nonblocking/quic.rs

```diff
@@ -1226,6 +1296,9 @@ struct ConnectionEntry {
     peer_type: ConnectionPeerType,
     last_update: Arc<AtomicU64>,
     port: u16,
+    #[allow(dead_code)]
+    // We do not explicitly use it, but its drop is triggered when ConnectionEntry is dropped.
+    client_connection_tracker: ClientConnectionTracker,
```

Comment:
nit: instead of using the allow lint, call it _client_connection_tracker. This is typically what is done with PhantomData fields too